src/ru/skypro/Employee.java

```diff
@@ -0,0 +1,66 @@
+package ru.skypro;
+
+import java.util.Objects;
+
+public class Employee {
+    private static int count;
+    private int id;
+    private String fullName;
+    private int department;
+    private int salary;
+
+    public Employee(String fullName, int department, int salary) {
+        count++;
+        id = count;
```

Comment:
можно объединить
```suggestion
        id = ++count;
```

https://www.programiz.com/article/increment-decrement-operator-difference-prefix-postfix